cmake/PcapPlusPlusUninstall.cmake.in

```diff
@@ -0,0 +1,21 @@
+if(NOT EXISTS "@CMAKE_BINARY_DIR@/install_manifest.txt")
+  message(FATAL_ERROR "Cannot find install manifest: @CMAKE_BINARY_DIR@/install_manifest.txt")
+endif()
+
+file(READ "@CMAKE_BINARY_DIR@/install_manifest.txt" files)
+string(REGEX REPLACE "\n" ";" files "${files}")
+foreach(file ${files})
+  message(STATUS "Uninstalling $ENV{DESTDIR}${file}")
+  if(IS_SYMLINK "$ENV{DESTDIR}${file}" OR EXISTS "$ENV{DESTDIR}${file}")
+    exec_program(
```

Comment:
maybe use file(REMOVE ...) https://cmake.org/cmake/help/latest/command/file.html#remove

Reply:
Since we are not using any symbolic link and also there is no plan for it (#1164), we can simplify it.

Reply:
I think there are some symlinks for the .so files and the version.

We can leave it like that if it's not the same behaviour. Maybe add a comment to explain. 

Thanks 🙏

Reply:
Done.